SampleShared/Samples/AppRemoting/Scripts/AppRemotingSample.cs

```diff
@@ -122,18 +130,24 @@ private void Update()
                                 ? $"Stopped listening on {hostIp}:{listenPort}"
                                 : $"Listening to incoming connection on {hostIp}";
 
+            string text = "Status unknown...";
             switch (m_appRemotingMode)
             {
                 case AppRemotingMode.none:
-                    outputText.text = commonMessage;
+                     text = commonMessage;
```

Comment:
nit: looks like an extra space got in here somehow 😄 

```suggestion
                    text = commonMessage;
```